packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -29,6 +29,8 @@ export const messages = {
     'The local development server was not able to start.',
   force_lightning_lwc_start_exited:
     'The local development server exited unexpectedly with code %s.',
+  force_lightning_lwc_start_no_lwc_folder:
+    'No \'lwc\' directory found in path %s',
```

Comment:
```suggestion
    'No \'lwc\' directory found',
```

Reply:
@ekapner Can you weigh in here for the error message presented to the user?

Context: Customer tries to start code preview without having an lwc folder in their default package directory.

Reply:
@lcampos - I didn't see this until now - do I have time to work on the error message?  cc: @LisandroFernandezSF